web/book/highlight-prql.js

```diff
@@ -72,7 +72,7 @@ formatting = function (hljs) {
 
   const CHAR_ESCAPE = {
     scope: "char.escape",
-    match: /\\\\|\\([bfnrt]|u\d{4})|x[0-9A-Fa-f]{2}/,
+    match: /\\\\|\\([bfnrt]|u\d{4})|U00[0-9A-Fa-f]{6}|x[0-9A-Fa-f]{2}/,
```

Comment:
Unrelated — but `u\d{4}` looks like it should be using hex rather than digits...

Reply:
Yeah, I think so too.

Reply:
I just filed PR #3570 to fix this.